From a9f23f088794c9b92c1471fc86c2b3c39ab475fd Mon Sep 17 00:00:00 2001
From: Stefan Hajnoczi <stefanha@redhat.com>
Date: Tue, 9 Mar 2021 14:19:13 +0000
Subject: [PATCH 2/2] spec: add explicit build dependency on make

Fedora 34 is removing make from the buildroot. An explicit dependency is
now required:
https://fedoraproject.org/wiki/Changes/Remove_make_from_BuildRoot

Adding an explicit build dependency on make seems reasonable across all
rpm-based distros. It won't hurt on distros where make is always
available in the buildroot.

Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
Link: https://lore.kernel.org/r/20210309141913.262131-3-stefanha@redhat.com
Signed-off-by: Jens Axboe <axboe@kernel.dk>
---
 liburing.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/liburing.spec b/liburing.spec
index 8607074..0268d23 100644
--- a/liburing.spec
+++ b/liburing.spec
@@ -7,6 +7,7 @@ Source0: https://brick.kernel.dk/snaps/%{name}-%{version}.tar.gz
 Source1: https://brick.kernel.dk/snaps/%{name}-%{version}.tar.gz.asc
 URL: https://git.kernel.dk/cgit/liburing/
 BuildRequires: gcc
+BuildRequires: make
 
 %description
 Provides native async IO for the Linux kernel, in a fast and efficient
-- 
2.30.2

